blocks/library/pullquote/index.js

```diff
@@ -145,9 +143,7 @@ registerBlockType( 'core/pullquote', {
 
 			return (
 				<blockquote className={ `align${ align }` }>
-					{ value && value.map( ( paragraph, i ) =>
-						<p key={ i }>{ paragraph.children && paragraph.children.props.children }</p>
-					) }
+					{ value.map( ( paragraph ) => paragraph.children ) }
```

Comment:
It's unfortunate that this still means that the block author needs to be aware of the editable value structure.

Reply:
Actually, in this case this is the structure from the block attribute's `query` result, necessary because there's no wrapper to separate content from the citation that would allow us to use the `children` source directly.